tests/tracking/test_fluent.py

```diff
@@ -476,62 +492,64 @@ def test_search_runs_no_arguments():
     mock_experiment_id = mock.Mock()
     experiment_id_patch = mock.patch("mlflow.tracking.fluent._get_experiment_id",
                                      return_value=mock_experiment_id)
-    get_paginated_runs_patch = mock.patch('mlflow.tracking.fluent._get_paginated_runs',
-                                          return_value=[])
+    get_paginated_runs_patch = mock.patch('mlflow.tracking.fluent._paginate', return_value=[])
```

Comment:
I don't like the changes in this test, as I don't feel like they are actually testing anything really, at least not really what the name implies, but I couldn't figure out a good way to validate a lambda short of what I added, which only validates that it was a callable, not anything about it. If you have any ideas of how to improve this please let me know.

Reply:
Wow you're right, this is a complicated test to write. 

I know this is lame, but what do you think of just asserting that `_get_experiment_id` was called? I think this does validate the behavior the test is trying to verify. (If you decide to do this, you should also update the docstring to ` When no experiment ID is specified, it should try to get the implicit one`.)

Reply:
IMO you should actually remove this test, since it's not really testing anything except that the lambda gets called.